databases/backends/postgres.py

```diff
@@ -128,6 +128,12 @@ def __iter__(self) -> typing.Iterator:
     def __len__(self) -> int:
         return len(self._row)
 
+    def __str__(self) -> str:
+        return self._row.__str__()
+
+    def __repr__(self) -> str:
+        return self._row.__repr__()
```

Comment:
I am not insisting here but this goes closer to the nature of `repr`:

```suggestion
        return f"Record({repr(self._row)}, {repr(self._result_columns)}, {repr(self._dialect)})"
```